### PR TITLE
ref: Use `npm` instead of `yarn`

### LINKED ISSUE #106: Use `npm` instead of `yarn`

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Personal preference here, figured it'd be nice to match Honeycomb as well.

- Deletes `yarn.lock`
- Runs an `npm install`
- Updates the README to use npm commands
- Updates the actions to use npm commands

>[!NOTE]
> This includes the PRs #103 #104 #105 and #107 to prevent merge conflicts down the line. Those should be merged first!

### CODE REVIEW

File: docs/project_organization/directory_structure.mdx [hunk 1]
```diff
@@ -38,149 +38,184 @@ This folder contains different files used to pass environment variables (setting
...
 :::
 
 ## public/
 
 The `public` directory contains files that are used as assets in the built app.
 
 - `index.html` is the entry point of the website
-  - Changing `<title>Honeycomb</title>` will update the name you can see in the browser tab
+  - Changing `<title>Honeycomb</title>` will update the name you can see in the browser tab!
 - `favicon.ico` is the small icon you can see in the browser tab
```
Comment (eldu) [inline docs/project_organization/directory_structure.mdx]:
```suggestion
- `favicon.ico` is the small 16x16px icon you can see in the browser tab

```

File: docs/project_organization/directory_structure.mdx [hunk 2]
```diff
@@ -38,149 +38,184 @@ This folder contains different files used to pass environment variables (setting
...
-- `electron.js` contains all of the code related to the electron app
+- `manifest.json` contains metadata about the web app
```
Comment (eldu) [inline docs/project_organization/directory_structure.mdx]: Do we need to say something about this as not something people should touch?

File: docs/project_organization/directory_structure.mdx [hunk 3]
```diff
@@ -38,149 +38,184 @@ This folder contains different files used to pass environment variables (setting
...
+A [trial](https://www.jspsych.org/7.3/overview/timeline/#a-single-trial) is the base unit of an experiment. These trials are ordered into procedures and timelines to create the task itself.
 
-A trial is the base unit of a JsPsych experiment. Each trial should be its own file within this folder - the files in [src/timelines/](#timelines) will combine these trials into the full experiment.
-
-- `honeycombTrials.js` contains the individual trials used in the Honeycomb task. These trials are imported into `honeycombBlock.js` and `honeycombTimeline.js`.
-- `adjustVolume.js` prompts the user to adjust the volume on their computer.
+- `adjustVolume.js` prompts the user to adjust the volume on their tablet.
 - `camera.js` contains trials for beginning and ending a camera recording.
+- `conclusion.js` displays a message to the user indicating that the experiment has concluded.
+- `fixation` displays a fixation dot in the center of the screen. It contains additional logic for flashing a photodiode spot and emitting an event code based on the environment settings.
```
Comment (eldu) [inline docs/project_organization/directory_structure.mdx]: is it a js file?

```suggestion
- `fixation.js` displays a fixation dot in the center of the screen. It contains additional logic for flashing a photodiode spot and emitting an event code based on the environment settings.

```

**Review by eldu — APPROVED** [review]

File: docs/project_organization/directory_structure.mdx [hunk 4]
```diff
@@ -167,7 +167,8 @@ A [trial](https://www.jspsych.org/7.3/overview/timeline/#a-single-trial) is the
 - `adjustVolume.js` prompts the user to adjust the volume on their tablet.
 - `camera.js` contains trials for beginning and ending a camera recording.
 - `conclusion.js` displays a message to the user indicating that the experiment has concluded.
-- `fixation` displays a fixation dot in the center of the screen. It contains additional logic for flashing a photodiode spot and emitting an event code based on the environment settings.
+- `fixation.js` displays a fixation dot in the center of the screen. It contains additional logic for flashing a photodiode spot and emitting an event code based on the environment settings.
+
```
Comment (RobertGemmaJr) [inline docs/project_organization/directory_structure.mdx]:
```suggestion
```

**Pull request merged**